template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApp-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Microsoft",
+    "classifications": [ "Common", "Console" ],
```

Comment:
@KathleenDollard could you please review name, identities, description etc? They are mostly copied from console now.